GK-js-hw-02/task-06.js

```diff
@@ -0,0 +1,27 @@
+const sumElements = function(arr) {
+  let sum = 0;
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Велике прохання, не відключати перевірки лінтом) 
```suggestion
  for (let i = 0; i < arr.length; i += 1) {
```